### PR TITLE
287159 Code sample makes no sense

### PR DESCRIPTION
**Pull request by PeterSmithRedmond**
Fixed the specific issue, which is that the listener wasn't declared.  Used code from the full sample.

# Title
Updated C# and VB sample code to declare the listener

## Summary

User complained that the code didn't make sense because the listener was not declared.  Fixed by declaring the listener.

Fixes #287159

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/network-programming/listening-with-sockets.md [hunk 1]
```diff
@@ -48,11 +48,15 @@ IPEndPoint localEndPoint = new IPEndPoint(ipAddress, 11000);
...
   
 ```vb  
+Dim listener As New Socket(ipAddress.AddressFamily, _  
+    SocketType.Stream, ProtocolType.Tcp) 
 listener.Bind(localEndPoint)  
 listener.Listen(100)  
 ```  
   
 ```csharp  
+Socket listener = new Socket(ipAddress.AddressFamily,
```
Comment (stevewhims) [inline docs/framework/network-programming/listening-with-sockets.md]: I'd tend to declare as var here, since the type is obvious.

**Review by stevewhims — APPROVED** [review]
Looks good. But I'd use var, myself.

**Comment by PeterSmithRedmond** [comment]
The original code predates var ☺

Peter

**Comment by JRAlexander** [comment]
LGTM. Thanks @PeterSmithRedmond for your contribution! I'll merge this now.

**Review by JRAlexander — APPROVED** [review]
LGTM. Thanks @PeterSmithRedmond for your contribution! I'll merge this now.

**Pull request merged**